configs/beit/upernet_beit_large_fp16_8x1_640x640_160k_ade20k.py

```diff
@@ -0,0 +1,50 @@
+_base_ = [
+    '../_base_/models/upernet_beit.py', '../_base_/datasets/ade20k_640x640.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_320k.py'
+]
```

Comment:
inherit configs/beit/upernet_beit_base_8x2_640x640_160k_ade20k.py and revise little